iothub/device/src/Common/Security/SharedAccessSignatureBuilder.cs

```diff
@@ -33,10 +45,20 @@ public string Key
             }
         }
 
+        /// <summary>
+        /// The resource being accessed
+        /// </summary>
         public string Target { get; set; }
 
+        /// <summary>
+        /// Token expiry instant
```

Comment:
do you think the term "instant" might be misleading, since this is a timespan?

also, kind-of reminds me of this java type: https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html 😆

Reply:
Good point. How does "The time before the signature expires" sound?

Reply:
Why not `The time the token expires.`?